pkg/utils/util.go

```diff
@@ -46,6 +46,11 @@ func ContainsPod(list []string, element string) bool {
 	return false
 }
 
+func SkipSubResources(kind string) bool {
+	s := []string{"PodExecOptions", "PodAttachOptions", "PodProxyOptions", "ServiceProxyOptions", "NodeProxyOptions"}
```

Comment:
Can we add a comment here that we need to update this list for resources that doesn't have the Group/Version.